Use support.md instead of direct links to GitHub (#14648)
pingcap/docs: support.md

If you encounter a problem when you use TiDB, you can reach out for support from PingCAP or the TiDB community via the following methods:

+ Get support from PingCAP for [TiDB Enterprise Edition](https://www.pingcap.com/tidb-enterprise/):

    - [Submit a request](https://tidb.support.pingcap.com).

+ Get support from PingCAP for TiDB Cloud:

>     - Report an issue directly from the TiDB Cloud web interface.

Reviewer: ```suggestion
    - Report an issue directly from the [TiDB Cloud console](https://tidbcloud.com/).
```

    - [Documentation for TiDB Cloud support](https://docs.pingcap.com/tidbcloud/tidb-cloud-support)

+ Seek help from the TiDB community:

    - The [TiDB Forum](https://ask.pingcap.com/)
    - [Discord channels](https://discord.gg/DQZ2dy3cuc?utm_source=doc)
    - Slack channels: [#everyone](https://slack.tidb.io/invite?team=tidb-community&channel=everyone&ref=docs) (English), [#tidb-japan](https://slack.tidb.io/invite?team=tidb-community&channel=tidb-japan&ref=docs) (Japanese)
    - [Stack Overflow](https://stackoverflow.com/questions/tagged/tidb) (questions tagged with #tidb)

>     - [GitHub](https://github.com/pingcap/tidb/)

Reviewer 2: ```suggestion

+ Report a bug

    - Submit an issue [on GitHub](https://github.com/pingcap/tidb/issues/new/choose)
```

If you encounter a problem when you use TiDB, you can reach out for support from PingCAP or the TiDB community via the following methods:

+ Get support from PingCAP for [TiDB Enterprise Edition](https://www.pingcap.com/tidb-enterprise/):

>     - [Submit a request](https://tidb.support.pingcap.com).

Reviewer: ```suggestion
    - [Submit a request](https://tidb.support.pingcap.com)
```

+ Get support from PingCAP for TiDB Cloud:

>     - Report an issue directly from the [TiDB Cloud console](https://tidbcloud.com/).

Reviewer: ```suggestion
    - Report an issue directly from the [TiDB Cloud console](https://tidbcloud.com/)
```

    - Slack channels: [#everyone](https://slack.tidb.io/invite?team=tidb-community&channel=everyone&ref=docs) (English), [#tidb-japan](https://slack.tidb.io/invite?team=tidb-community&channel=tidb-japan&ref=docs) (Japanese)
    - [Stack Overflow](https://stackoverflow.com/questions/tagged/tidb) (questions tagged with #tidb)

+ Report a bug

    - Submit an issue [on GitHub](https://github.com/pingcap/tidb/issues/new/choose)

+ Learn TiDB's implementation and design

>     - [TiDB Development Guide](https://pingcap.github.io/tidb-dev-guide/)

Reviewer: ```suggestion
    - [TiDB development guide](https://pingcap.github.io/tidb-dev-guide/)
```

Author: Any specific for making this lowercase? On the actual devguide it is also spelled with capitals.

Reviewer: https://developers.google.com/style/capitalization#capitalization-in-references-to-titles-and-headings
> In references to any title or heading from a document that follows this guide, use sentence case even if the title or heading itself uses title case.
